app/src/styles/global.css

```diff
@@ -168,13 +168,16 @@ textarea {
       bottom: 0;
       box-shadow: 0 1px 0 0 rgba(0, 0, 0, 0.06), 0 2px 7px 0 rgba(0, 0, 0, 0.05);
       display: flex;
-      flex-direction: row-reverse;
       height: 60px;
       left: 0;
-      padding: 1em;
+      padding: 1em 4em 1em 2em;
```

Comment:
This isn't great.

I was thinking the `.form` should expand to take the full height of the `.drawer`, and enclose the `.footer` but that might be best picked up separately.